DigitalOffice/Services/TimeManagementService/Validators/ChangeWorkTimeRequestValidator.cs

```diff
@@ -0,0 +1,42 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.TimeManagementService.Models;
+using System;
+
+namespace LT.DigitalOffice.TimeManagementService.Validators
+{
+    public class ChangeWorkTimeRequestValidator : AbstractValidator<ChangeWorkTimeRequest>
+    {
+        public ChangeWorkTimeRequestValidator()
+        {
+            RuleFor(time => time.Id)
+                   .NotEmpty()
+                   .WithMessage("Id is empty.");
+
+            RuleFor(time => time.WorkerUserId)
+                    .NotEmpty()
+                    .WithMessage("WorkerUserId is empty.");
+
+            RuleFor(date => date.StartTime)
+                .NotEqual(new DateTime());
+
+            RuleFor(wt => wt.EndTime)
+                .NotEqual(new DateTime());
+
+            RuleFor(time => time.Title)
+                    .NotEmpty()
+                    .WithMessage("Title is empty.")
```

Comment:
Can be removed.

Reply:
I do not think so

Reply:
https://digitaloffice-group.slack.com/archives/C016H4Z2LGL/p1595663778001900

Reply:
fixed